js/controller.js

```diff
@@ -1209,14 +1217,21 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.mb.publish(OO.EVENTS.SET_CURRENT_AUDIO, this.state.currentVideoId, currentTrack);
     },
 
-    onSeeked: function() {
+    onSeeked: function(event, time) {
       this.state.seeking = false;
       if (this.state.queuedPlayheadUpdate) {
         OO.log('popping queued update');
         this.skin.updatePlayhead.apply(this.skin, this.state.queuedPlayheadUpdate).then(() => {
           this.state.queuedPlayheadUpdate = null;
         }).catch(() => {OO.log('onSeeked: Could not set new state for skin')});
       }
+      if (this.state.cast.isReceiver) {
+        this.skin.updatePlayhead(time);
+        if (this.state.playerState === CONSTANTS.STATE.PLAYING) {
```

Comment:
How sure are we that the player state will be always equals to "playing" after a seek event to restrict the visibility of the control bar based on that state?

Reply:
I hope so) I checked it a couple of times. When video is playing and we move playhead along the scrubber bar the  _playerState_ remains the same